### ISSUE TITLE
Reconsider the gear menu

### ISSUE THREAD

**Issue by ghost** [mozilla/testpilot#2867]

**Comment by ghost** [comment]
Let's move things out that we want to get more traction like Discourse

**Comment by Pekochen** [comment]
Hi @wresuolc Can you please describe in more detail the gear menu you want us to design or what's the problem we face? Thanks!

**Comment by ghost** [comment]
Sorry @Pekochen .  We must have filed this in the middle of triaging.  I try to be better about descriptions.

We were talking about the current Settings menu in the top right of the test pilot site.  It's got a few sub-bullets right now (wiki, discourse, github, uninstall).  We wanted to move Discourse to be a top level link like the blog, but I marked it needs:product for any other decisions -- it sounds like we were talking about getting rid of the settings menu altogether from the way this was worded.

@johngruen - what should be done here?

**Comment by fzzzy** [comment]
If I remember correctly, we just wanted to get rid of the settings menu entirely, and put all the options currently in the settings menu at the top level.

**Comment by ChenMorpheus** [comment]
@clouserw @johngruen @fzzzy I agree it's odd to put all these options in Settings. However, we might need to take a moment to consider if it's appropriate to display all options at the top level. My concerns are: are these items so essential that they must place here? Will uninstall button encourages users to uninstall? 

Therefore, I created the wireframe below to propose two solutions. Proposal A is to list all options and use an icon to differentiate the button of "uninstall Test Pilot." 

Proposal B is to collect these options in the footer with the current items and provide an organized structure. For the top level, I would suggest examing the CTR of the options in Settings to decide which one should stay there. 

![default](https://user-images.githubusercontent.com/18019946/31765257-642dc3c0-b4f6-11e7-80b4-61a7758b76b8.png)


What do you think? Let me know if any thoughts, thanks.

**Comment by SoftVision-PaulOiegas** [comment]
Personal opinion. I think B is to mainstream right now, every website has the footer like this. I think A looks better because its more slimmer. But I would suggest moving the uninstall button on the footer as in proposal B, we don't want to show that so soon maybe.
Also, I think we do want to have at least the "Blog", "Discourse" and "File a bug" on top. 

Maybe it would be a good idea to have some of the top buttons also on the experiments details pages and have them linked to experiment Discourse and new issue buttons. Maybe that we'll help the users to discuss topics and log issues into the right page/repo.

**Comment by lmorchard** [comment]
> I would suggest examing the CTR of the options in Settings to decide which one should stay there.

FWIW, the current CTR for all of these things is very low. That's the reason for this issue: We'd like to *increase* the CTR on at least some of these things by making them more visible. That is, encourage more users to visit the discourse forums, get more readers of the blog, etc. I guess the thing is to decide which of these are most important to push for more a higher CTR

**Comment by ChenMorpheus** [comment]
@lmorchard @SoftVision-PaulOiegas Thanks for the information. If that's the case, I would suggest displaying "news feed," "blog," "discourse" and "file a bug" at the top level. The news feed is a proposed change for the UX of the featured experiment (https://github.com/mozilla/testpilot/issues/2976). Also, I would want to make "file a bug" into an action-emphasized button since it provides different purpose compared to other options and this might encourage users to file more bugs. 

As for footer, I would put wiki link here unless you think it needs much attention. The uninstall button staying at the right bottom corner makes more sense to me since it represents the closure to Test Pilot for users. @johngruen What do you think?

![default](https://user-images.githubusercontent.com/18019946/31871005-be3f0316-b7e3-11e7-942a-76c43baf3ac1.png)

**Comment by SoftVision-PaulOiegas** [comment]
Looks great! Just one thing, I don't see "Legal" and "Give Feedback" links on bottom.

**Comment by ChenMorpheus** [comment]
So sorry that I didn't notice they weren't on the footer. Updated, thanks for your excellent eyes. Please let me know if the order is inappropriate. 

BTW, I would like to propose to display the buttons as well for the moment users haven't installed Test Pilot. When users scroll down the page, the toolbar stays at the top level and shows "install Test Pilot" button.

![default](https://user-images.githubusercontent.com/18019946/31871656-9cc71c0a-b7e8-11e7-8b47-17181a357020.png)

**Comment by johngruen** [comment]
I like option B best w/ the scrolled state featured in the most recent comment. I'm a little averse to having Discourse up there for users without the add-on installed though. Seems risky to bounce people out of the TxP UI before they've installed.

**Comment by ChenMorpheus** [comment]
@johngruen Since I got the request to align with [AMO's new design](https://addons-dev.allizom.org/en-US/android/), I've updated the screens to provide various scenarios. You can find the link below for the different cases, and I also attach several screens for clarification. Please let me know if any questions, thanks. 
(The Turquoise bar at the top level is just the navigation for this prototype. It's not belonged to the content.)

https://mozilla.invisionapp.com/share/QCCPNI3YZ#/260183664_1-0_Not_Firefox_Not_Installed_TxP

![2 0 not firefox_not installed txp_sd](https://user-images.githubusercontent.com/18019946/31984218-3bae3212-b993-11e7-8122-74f40672689d.png)
(The install button only displays when users scroll down)


![6 0 txp installed_dp](https://user-images.githubusercontent.com/18019946/31984451-374b4f10-b994-11e7-827b-e99b38554b41.png)
(Removing the "Discourse" and "File a bug" in detail page is to prevent confusion with the experiment's links.)

**Comment by ChenMorpheus** [comment]
We had a meeting with Design System team and realized that we must put links for Firefox products in the footer. (You can find attachment for example)So we will sync with them and update the screen again and provide measurement accordingly. 

![default](https://user-images.githubusercontent.com/18019946/32096584-76c85e8e-bb39-11e7-9d31-13263e3e1db2.png)

**Comment by johngruen** [comment]
@johngruen to provide some have-to-haves in the footer.

**Comment by johngruen** [comment]
@ChenMorpheus as long as we keep the current test pilot links i am happy!

**Comment by ChenMorpheus** [comment]
Please find the the prototype for various updated use cases and the links below for measurements and assets.

![image](https://user-images.githubusercontent.com/18019946/33549772-a27eaf50-d926-11e7-9967-2fe34fc6632b.png)

![image](https://user-images.githubusercontent.com/18019946/33549781-ab67c7aa-d926-11e7-9b3f-fd336faaee46.png)


Prototype: 
https://mozilla.invisionapp.com/share/QCCPNI3YZ#/260183664_1-0_Not_Firefox_Not_Installed_TxP

Measurements: 
https://mozilla.github.io/testpilot-assets/Test_Pilot_Website/01_Measurements/Photonized_Txp_Website_Final/

**Comment by johngruen** [comment]
TODO: figure out what CTA for installed users should be

**Comment by ChenMorpheus** [comment]
@johngruen We could simply not provide CTA for installed users, ie. remove the "file a bug" in the detail page below. 
![group 3](https://user-images.githubusercontent.com/18019946/34765984-ec13dcf2-f62d-11e7-965a-c4556cc395c5.png)
